lib/mini-lightbox.js

```diff
@@ -1,6 +1,45 @@
 // MiniLightbox
 // </> with <3 by Ionică Bizău
-(function (root) {
+(function () {
```

Comment:
So, we can remove this wrapping function, including the first two lines (the commented ones). :fire: